pkg/trace/stats/client_stats_aggregator.go

```diff
@@ -44,6 +43,7 @@ type ClientStatsAggregator struct {
 	oldestTs      time.Time
 	agentEnv      string
 	agentHostname string
+	conf          *config.AgentConfig
```

Comment:
I think that just passing `agentVersion` here makes a lot of sense when paired with the two fields above 🙂